### PR TITLE
Update to stacks js

### PR DESCRIPTION
**Pull request by yknl**
This PR removes dependency on `blockstack.js` and switches to the new `stacks.js` packages: `@stacks/auth`, `@stacks/storage` and `@stacks/profile`.

The following packages will be updated after they're published to `@stacks`:
`@blockstack/connect`
`@blockstack/prettier-config`
`@blockstack/eslint-config`
`@blockstack/ui`

### CODE REVIEW

**Comment by timstackblock** [comment]
Please let me know when this is ready for test and can be moved to the QA column

File: package.json [hunk 1]
```diff
@@ -14,14 +14,18 @@
...
-    "url": "https://github.com/blockstack/blockstack-todos"
+    "url": "https://github.com/blockstack/stacks-todos"
   },
-  "version": "0.1.0",
+  "version": "0.2.0",
   "private": true,
   "dependencies": {
-    "@blockstack/connect": "^2.7.7",
+    "@blockstack/connect": "^2.12.7",
     "@blockstack/ui": "^2.7.0",
```
Comment (agraebe) [inline package.json:23]: can we also update to https://www.npmjs.com/package/@stacks/ui ?

File: package.json [hunk 2]
```diff
@@ -14,14 +14,18 @@
...
+    "@stacks/auth": "^1.0.0-beta.7",
+    "@stacks/profile": "^1.0.0-beta.7",
+    "@stacks/storage": "^1.0.0-beta.7",
     "blockstack": "git://github.com/blockstack/blockstack.js.git#3124e05",
+    "@blockstack/stacks-transactions": "0.7.0",
```
Comment (agraebe) [inline package.json:28]: and I assume this would become `@stacks/transactions` ?

File: package.json (hunk 1 above)
Comment (yknl) [reply package.json:23]: `@stacks/ui` is not ready for use yet, we'll be using the `@blockstack/ui` for the time being

File: package.json (hunk 2 above)
Comment (yknl) [reply package.json:28]: The todo app itself actually doesn't need to import the transaction lib, but the lib is declared as a peer dependency of Connect so removing this line breaks the build. We'll need to fix this in Connect.

Comment (hstove) [reply package.json:28]: I'm not sure how we can get around this, as Connect uses many types from `stacks-transactions` in the TX signing methods.

**Comment by agraebe** [comment]
I noticed you need to include a new `package-lock.json`

**Review by agraebe — APPROVED** [review]

**Pull request merged**